docs/config.md

```diff
@@ -102,6 +102,8 @@ Here are the available settings:
 
 - `use_value`: The value of the parameter to be used by Scribe to authenticate response calls. This will **not** be included in the generated documentation. If this value is null, Scribe will use a random value.
 
+- `display_value`: The value of the parameter that will be included in the generated documentation. If this value is null, Scribe will use a random value.
```

Comment:
```suggestion
- `placeholder`: Placeholder your users will see for the auth parameter in the example requests. If this is empty, Scribe will generate a realistic-looking auth token instead. Defaults to: "{YOUR_AUTH_KEY}".
```

(I'm switching "if this is null" to "if this is empty" in the documentation generally, because it's both more technically correct and less of an implementation detail.)